tests/resources/VCH-Util.robot

```diff
@@ -79,8 +79,14 @@ Set Test Environment Variables
     Wait Until Keyword Succeeds  5x  1s  Create Unique Bridge Network
 
 Create Unique Bridge Network
+    # Ensure unique bridges are non-overlapping in a shared build environment (our CI)
+    @{URLs}=  Split String  %{TEST_URL_ARRAY}
+    ${idx}=  Get Index From List  ${URLs}  %{TEST_URL}
+    ${lowerVLAN}=  Evaluate  (${idx}+2) * 100
```

Comment:
This would be more flexible injected from where ever the TESTURL is injected. This will require local environments have vlan ID 200-300 configured for use.

Reply:
local environments shouldn't need anything configured AFAIK?  The bridge network doesn't require any services running on it at all and is only used internally per VCH, so essentially the only thing we need on local environments is that nothing else is running on that vlan and that we can create portgroups with govc otherwise this logic should work fine I think.